fling/model/moon_models.py

```diff
@@ -0,0 +1,63 @@
+from torch import nn
```

Comment:
Don't create a new model class. Reuse the previous resnet. For example, you can add a new argument ``mode`` to the previous  forward function. Modes can contain: ['compute-logit', 'compute-feature-logit'], and the default value is 'compute-logit'